roofit/doc/developers/roofit_ad.md

```diff
@@ -505,7 +567,7 @@ output using the `buildCall()` function.
 ```
 
 \note The implementation of the `buildCallToAnalyticIntegral()` function is
```

Comment:
Also at least the first time you mention `buildCallToAnalyticIntegral()`, you should fully qualify the name, including the class name `RooAbsReal::`.